pkg/apiserver/apiserver.go

```diff
@@ -98,6 +98,45 @@ func (s *Service) IsRunning() bool {
 	return s.status.IsRunning()
 }
 
+var Modules = fx.Options(
+	fx.Provide(
+		newAPIHandlerEngine,
+		newClients,
+		dbstore.NewDBStore,
```

Comment:
In test scenarios you may like a memory local store instead. The memory sqlite is widely used in unit tests and it is reliable enough.